src/rbc_validator.c

```diff
@@ -2,42 +2,41 @@
 // Created by cp723 on 2/7/2019.
 //
 
+#include <openssl/err.h>
+#include <openssl/evp.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
 
-#include <openssl/evp.h>
-#include <openssl/err.h>
-
 #if defined(USE_MPI)
 #include <mpi.h>
+
 #include "cmdline/cmdline_mpi.h"
 #else
 #include <omp.h>
+
 #include "cmdline/cmdline_omp.h"
 #endif
 
-#include "validator.h"
 #include "crypto/cipher.h"
 #include "crypto/ec.h"
 #include "crypto/hash.h"
-#include "seed_iter.h"
 #include "perm.h"
-#include "uuid.h"
+#include "seed_iter.h"
 #include "util.h"
+#include "uuid.h"
+#include "validator.h"
 
-#define ERROR_CODE_FOUND 0
-#define ERROR_CODE_NOT_FOUND 1
-#define ERROR_CODE_FAILURE 2
+enum StatusCode { SC_Found = 0, SC_NotFound = 1, SC_Failure = 2 };
 
 // If using OpenMP, and using Clang 10+ or GCC 9+, support omp_pause_resource_all
-#if !defined(USE_MPI) && ((defined(__clang__) && __clang_major__ >= 10) || (!defined(__clang) && \
-    __GNUC__ >= 9))
-#define OMP_DESTROY()\
-if(omp_pause_resource_all(omp_pause_hard)) {\
-    fprintf(stderr, "ERROR: omp_pause_resource_all failed.");\
-}
+#if !defined(USE_MPI) && \
```

Comment:
Kind of a weird split

Reply:
I think it's because the line it's too long, and it's avoiding splitting in the middle of the parentheses